taca_ngi_pipeline/deliver/deliver_grus.py

```diff
@@ -369,6 +370,28 @@ def add_supr_name_delivery_in_charon(self, supr_name_of_delivery):
             logger.error('Failed to update delivery_projects in charon while delivering {}. Error says: {}'.format(self.projectid, e))
             logger.exception(e)
 
+    def add_supr_name_delivery_in_statusdb(self, supr_name_of_delivery):
+        '''Updates delivery_projects in StatusDB at project level
+        '''
+        save_meta_info = getattr(self, 'save_meta_info', False)
+        if not save_meta_info:
+            return
+        status_db = statusdb_session(self.config_statusdb)
+        project_page=json.loads(json.dumps(status_db.get_project(self.projectid)))
```

Comment:
is there a good reason for the dump and load (instead of just using the returned object)?

Reply:
I've removed it!